docs/content/v2024.2/admin/yb-admin.md

```diff
@@ -1779,6 +1779,12 @@ A successful operation of the above command returns a message with a DB stream I
 CDC Stream ID: d540f5e4890c4d3b812933cbfd703ed3
 ```
 
+{{ < note title="Note" > }}
+
+It is recommended that you create a stream ID with `EXPLICIT` checkpointing mode only, which is also the default checkpointing mode now. The `IMPLICIT` checkpointing mode will be completely removed in future releases.
+
+{{ < /note > }}
+
```

Comment:
Not rendering

Reply:
Also, could we put the `deprecation notice` as title instead of the `note`? @aishwarya24

Reply:
This hasn't been deprecated yet so in my opinion, we should keep it like this. But I can make the change if it makes better sense to put as `Deprecation notice`